src/components/notifications/VoNotificationsBanner.vue

```diff
@@ -107,9 +107,7 @@
   async function onClick (e: MouseEvent | KeyboardEvent) {
     if (!banner.value) return
 
-    onClose()
-
-    await nextTick()
+    setTimeout(() => onClose(), 500)
```

Comment:
I dont love using a timeout here, but it was the only way I could get it to not call the link before it was updated.